src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -70,7 +70,7 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
     $definition = \Drupal::entityTypeManager()->getDefinition($target_type);
 
     if ($bundle_key = $definition->getKey('bundle')) {
-      $bundles = Og::groupTypeManager()->getAllGroupBundles($target_type);
+      $bundles = Og::groupTypeManager()->getGroupsForEntityType($target_type);
```

Comment:
In #404 ::getGroupsForEntityType() was renamed to ::getGroupBundleIdsByEntityType():

```suggestion
      $bundles = Og::groupTypeManager()->getGroupBundleIdsByEntityType($target_type);
```